.ci/scripts/run-test-in-docker.sh

```diff
@@ -23,6 +23,7 @@ fi
 docker run \
   $DOCKER_RUN_OPTIONS \
   -u '0:0' \
-  -e NPM_COMMAND=${1:-''} \
+  -v "$(pwd):/synthetics-recorder" \
+  -e NPM_COMMAND=${1:-'ci'} \
```

Comment:
This one was causing the `npm ci` to be skipped since no params were passed and hence `''` was the default value

Reply:
AFAIR `npm ci` was already run from one of the previous steps so we didn't need to run it again

Reply:
That's correct, I'll remove it then